Gopkg.lock

```diff
@@ -384,11 +384,25 @@
 
 [[projects]]
   branch = "openshift-4.2-cluster-api-0.1.0"
-  digest = "1:efca7e07cbca01b615c1f9057da17b10ea6b7e43e263226a3ef04eebe16f34cf"
```

Comment:
Can we please have a proper commit message here: e.g "Revendor for fetching cluster-api informers" or so